Samples/ExtendedControlAndMetadata/MediaCaptureSampleExtendedControl/MainPage.xaml.cpp

```diff
@@ -0,0 +1,786 @@
+﻿//*********************************************************
+//
+// Copyright (c) Microsoft. All rights reserved.
+// THIS CODE IS PROVIDED *AS IS* WITHOUT WARRANTY OF
+// ANY KIND, EITHER EXPRESS OR IMPLIED, INCLUDING ANY
+// IMPLIED WARRANTIES OF FITNESS FOR A PARTICULAR
+// PURPOSE, MERCHANTABILITY, OR NON-INFRINGEMENT.
+//
+//*********************************************************
+
+//
+// MainPage.xaml.cpp
+// Implementation of the MainPage class.
+//
+
+#include "pch.h"
+#include "MainPage.xaml.h"
+#include "ppl.h"
+#include "CustomExtendedProperty.h"
+
+using namespace MediaCaptureSampleExtendedControls;
+
+using namespace Windows::System;
+using namespace Windows::Foundation;
+using namespace Windows::Foundation::Collections;
+using namespace Platform;
+using namespace Windows::UI;
+using namespace Windows::UI::Core;
+using namespace Windows::UI::Xaml;
+using namespace Windows::UI::Xaml::Controls;
+using namespace Windows::UI::Xaml::Navigation;
+using namespace Windows::UI::Xaml::Data;
+using namespace Windows::UI::Xaml::Media;
+using namespace Windows::UI::Xaml::Media::Imaging;
+using namespace Windows::Storage;
+using namespace Windows::Media;
+using namespace Windows::Media::MediaProperties;
+using namespace Windows::Media::Capture;
+using namespace Windows::Media::Capture::Frames;
+using namespace Windows::Storage::Streams;
+
+using namespace concurrency;
+using namespace CustomExtendedProperties;
+
+Platform::String^ ExposureControlID = L"{1CB79112-C0D2-4213-9CA6-CD4FDB927972} 12"; //KSCameraExtendedControlPropSet and ExposurePropSetID (see KSPROPERTY_CAMERACONTROL_EXTENDED_EXPOSUREMODE in ksmedia.h)
```

Comment:
>Platform::String^ ExposureControlID [](http://example.com/codeflow?start=0&length=35)

are we using c++/cx?

Reply:
yes we want to avoid using C++/CLI in the future. I fixed this sample and did not convert it, we can do so in another future change if we feel like it.